go/vt/vtadmin/api_test.go

```diff
@@ -23,17 +23,26 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"vitess.io/vitess/go/vt/grpcclient"
+	"vitess.io/vitess/go/vt/topo/memorytopo"
 	"vitess.io/vitess/go/vt/vitessdriver"
 	"vitess.io/vitess/go/vt/vtadmin/cluster"
 	"vitess.io/vitess/go/vt/vtadmin/cluster/discovery/fakediscovery"
 	"vitess.io/vitess/go/vt/vtadmin/grpcserver"
 	"vitess.io/vitess/go/vt/vtadmin/http"
+	vtadminvtctldclient "vitess.io/vitess/go/vt/vtadmin/vtctldclient"
```

Comment:
This is, _ahem_, admittedly extremely gross. Please help lol

Reply:
We could call it `vtctldclient` given that we're already in a "vtadmin" context. 🧐 Up to you though, I don't think this is any worse than all the other vtctld-related consonant salads lol.

Reply:
It conflicts with the import on line 40

Reply:
w e l p :')